OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -484,7 +484,10 @@ public interface ITargetablePositions
 		bool AlwaysEnabled { get; }
 	}
 
-	public interface IMoveInfo : ITraitInfoInterface { }
+	public interface IMoveInfo : ITraitInfoInterface
+	{
+		Color GetTargetLineColor();
```

Comment:
Would a property be better?

Reply:
This is consistent with how similar cases are handled throughout the code, e.g. in `AircraftInfo`.

Reply:
Ok.